File: SRM.Domain/Data/CongBoData.cs
using SRM.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SRM.Domain.Data
{
    public class CongBoData : ModelBase
    {
        public Guid? NoiDangBaoId { get; set; }
        public string? TenNoiDangBao { get; set; }
        public string? LinkMinhChungTapChi { get; set; }
        public string? Ten { get; set; }
        public string? DiaDiem { get; set; }
        public string? TenTapChi { get; set; }
        public string? NhaXuatBan { get; set; }
        public string? FileMinhChungBaiBao { get; set; }
        public string? LinkBaiBao { get; set; }
        public DateTime? NgayGuiDang { get; set; }
        public DateTime? NgayCongBo { get; set; }
        public decimal? ChiSoTacDong { get; set; }
        public int? Ky { get; set; }
        public int? Tap { get; set; }
        public string? Trang { get; set; }
        public decimal? DiemHoiDong { get; set; }
        public string? TenHoiDong { get; set; }
        public string? LoaiQ { get; set; }
        public int? ThanhQuaId { get; set; }
        public string? LinkMinhChungLoaiQ { get; set; }
        public string? TacGiaChinh { get; set; }
        public string? TacGiaLienHe { get; set; }
        public string? DongTacGias { get; set; }
        public string? LoaiHoTroChiPhi { get; set; }
        public string? PhanChiaSuDongGop { get; set; }
    }
}

File: srm.client/src/models/CongBoData.js
import ModelBase from './ModelBase';

/**
 * @typedef {Object} CongBoData
 * @property {string} [id] - Unique identifier
 * @property {string} [ngayTao] - Creation date
 * @property {string} [ngaySua] - Modification date
 * @property {string} [noiDangBaoId] - ID of the publication venue
 * @property {string} [tenNoiDangBao] - Name of the publication venue
 * @property {string} [linkMinhChungTapChi] - Link to journal verification
 * @property {string} [ten] - Title
 * @property {string} [diaDiem] - Location
 * @property {string} [tenTapChi] - Journal name
 * @property {string} [nhaXuatBan] - Publisher
 * @property {string} [fileMinhChungBaiBao] - Article verification file
 * @property {string} [linkBaiBao] - Link to the article
 * @property {string} [ngayGuiDang] - Date of submission
 * @property {string} [ngayCongBo] - Date of publication
 * @property {number} [chiSoTacDong] - Impact factor
 * @property {number} [ky] - Issue number
 * @property {number} [tap] - Volume
 * @property {string} [trang] - Pages
 * @property {number} [diemHoiDong] - Council score
 * @property {string} [tenHoiDong] - Council name
 * @property {string} [loaiQ] - Q-type classification
 * @property {number} [thanhQuaId] - Achievement ID
 * @property {string} [linkMinhChungLoaiQ] - Link to Q-type verification
 * @property {string} [tacGiaChinh] - Main author
 * @property {string} [tacGiaLienHe] - Corresponding author
 * @property {string} [dongTacGias] - Co-authors
 * @property {string} [loaiHoTroChiPhi] - Type of cost support
 * @property {string} [phanChiaSuDongGop] - Contribution distribution
 */

/**
 * Publication data model
 */
const CongBoData = {
  ...ModelBase,
  noiDangBaoId: null,
  tenNoiDangBao: null,
  linkMinhChungTapChi: null,
  ten: null,
  diaDiem: null,
  tenTapChi: null,
  nhaXuatBan: null,
  fileMinhChungBaiBao: null,
  linkBaiBao: null,
  ngayGuiDang: null,
  ngayCongBo: null,
  chiSoTacDong: null,
  ky: null,
  tap: null,
  trang: null,
  diemHoiDong: null,
  tenHoiDong: null,
  loaiQ: null,
  thanhQuaId: null,
  linkMinhChungLoaiQ: null,
  tacGiaChinh: null,
  tacGiaLienHe: null,
  dongTacGias: null,
  loaiHoTroChiPhi: null,
  phanChiaSuDongGop: null
};

export default CongBoData;